src/javascript/app/ng-wig/ngWigDirective.js

```diff
@@ -20,10 +21,11 @@ angular.module('ngWig')
         scope.execCommand = function (command, options) {
           if (command === 'createlink') {
             options = prompt('Please enter the URL', 'http://');
```

Comment:
I also have some thoughts about it, but we have scope inside. what would be the elegant way to do it?